CMakeLists.txt

```diff
@@ -120,6 +126,10 @@ FetchContent_MakeAvailable(repo-common repo-core repo-backend)
 #
 configure_file(src/libtriton_openvino.ldscript libtriton_openvino.ldscript COPYONLY)
 
+if(TRITON_BUILD_OPENVINO_VERSION MATCHES "^2021.2")
```

Comment:
Can you just create a TRITON_OPENVINO_VERSION def from TRITON_BUILD_OPENVINO_VERSION to be more general. And use target_compile_definition like here to add the define only to a specific target: https://github.com/triton-inference-server/tensorrt_backend/blob/main/CMakeLists.txt#L141